### PR TITLE
Add Platon Support

### PR DESCRIPTION
**Pull request by benben2019**
## Description

Add Platon support.

## Testing instructions

Test in the same manager as KCC was tested.

## Types of changes

New feature (non-breaking change which adds functionality) 

## Checklist

- [ ] Prefix PR title with `[WIP]` if necessary.
- [x] Add tests to cover changes as needed.
- [x] Update documentation as needed.
- [x] I have read the [guidelines](https://developer.trustwallet.com/wallet-core/newblockchain#integration-criteria) for adding a new blockchain
- [ ] If there is a related Issue, mention it in the description (e.g. Fixes #<issue_number> ).

### CODE REVIEW

**Review by catenocrypt — CHANGES_REQUESTED** [review]

File: src/proto/Platon.proto [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+syntax = "proto3";
+
+package TW.Platon.Proto;
+option java_package = "wallet.core.jni.proto";
+
+// TODO: typical balance transfer, add more fields needed to sign
+message TransferMessage {
```
[the remark this answers is not in the document]
Comment (benben2019) [reply src/proto/Platon.proto]: Platon.proto is no need. just reused Eth code.

File: swift/Podfile.lock [hunk 2]
```diff
@@ -13,4 +13,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: aac2324ba35cdd5631cb37618cd483887bab9cfd
 
-COCOAPODS: 1.11.2
+COCOAPODS: 1.11.3
```
[the remark this answers is not in the document]
Comment (benben2019) [reply swift/Podfile.lock]: Sorry about this.

File: docs/registry.md [hunk 3]
```diff
@@ -44,6 +44,7 @@ This list is generated from [./registry.json](../registry.json)
 | 461     | Filecoin         | FIL    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/filecoin/info/logo.png" width="32" />     | <https://filecoin.io/>        |
 | 474     | Oasis            | ROSE   | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/oasis/info/logo.png" width="32" />        | <https://oasisprotocol.org/>  |
 | 483     | Bluzelle         | BLZ    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/bluzelle/info/logo.png" width="32" />     | <https://bluzelle.com>        |
+| 486     | Platon           | LAT    | <img src="https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/platon/info/logo.png" width="32" />       | <https://www.platon.network/en> |
```
[the remark this answers is not in the document]
Comment (benben2019) [reply docs/registry.md:47]: OK, I submitted a PR: https://github.com/trustwallet/assets/pull/19918

**Comment by hewigovens** [comment]
Thanks for your interests, we will consider adding it later

**Comment by benben2019** [comment]
> Thanks for your interests, we will consider adding it later

Now we decide to pay the fee, would you please reopen this PR?  Thanks! 
And the logo & info PR #https://github.com/trustwallet/assets/pull/19918

**Pull request closed without merging**